str_analysis/generate_gnomad_json.py

```diff
@@ -859,7 +814,8 @@ def main():
     most_common_motif_lookup = compute_most_common_motif_lookup_dict(df)
     add_motif_classification_field(gnomad_json, most_common_motif_lookup)
     readviz_paths_to_rename, readviz_json = add_histograms_and_compute_readviz_paths(df, gnomad_json, most_common_motif_lookup)
-    remove_readviz_filenames_that_dont_exist(args, readviz_json)
+    if args.existing_readviz_filename_list:
+        remove_readviz_filenames_that_dont_exist(args, readviz_json)
```

Comment:
Good question. I run this script 2x. The 1st time is without this arg so it can generate the `readviz_rename_list__...tsv`. Then I use that output to rename the readviz image files to their encrypted filenames (so they can be made public without revealing the sample ids). Then I generate the --existing-readviz-filenames-list based on the image files that actually exist (rather than are expected to exist). Then I run this script a 2nd time with that input.